demo/thumbnail/basic/context.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
context is only created if you pass the flag `-C` when generating the files.
I was curious to know about this one since the component doesn't really need it because is not using anything from context.